<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>502</width>
    <height>418</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>502</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart_Calc_v2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(191, 191, 191);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 8px;
color:rgb(71, 71, 72);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="val_e">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Brackets">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Brackets2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sub">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mult">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Div">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pow">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>xʸ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>cos(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>sin(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>tan(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>acos(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>asin(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>atan(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>ln(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>log(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="val_x">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(128, 128, 128);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>80</width>
      <height>34</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {color:white;
    background-color:rgb(128, 128, 128);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(128, 128, 128);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>GRAF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(71, 71, 72);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>SmartCalc v2.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(191, 191, 191);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>Кредитный калькулятор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color:white;
    background-color:rgb(191, 191, 191);
    border-style: outset;
    border-width: 1px;
    border-radius: 8px;
    border-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>Депозитный калькулятор</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
